boards/nxp/imx95_evk/enetc_psi0.overlay

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
We may have the other overlay for other features, I suggest to put all of them in "dts" sub-directory just like what the other platforms are using.